django_socio_grpc/grpc_actions/actions.py

```diff
@@ -123,6 +124,11 @@ def __set_name__(self, owner, name):
             if "_decorated_grpc_action_registry" not in owner.__dict__:
                 owner._decorated_grpc_action_registry = {}
             owner._decorated_grpc_action_registry.update({name: self.get_action_params()})
+        # INFO - AM - 22/08/2024 - Send a signal to notify that a grpc action has been created. Used for now in cache deleter
+        grpc_action_set.send(sender=self, owner=owner, name=name)
```

Comment:
With this you access only the parent class which defines the action, this should be in the `register` method